ptz-explainer.md

```diff
@@ -177,6 +177,9 @@ Requesting the PTZ permission is gated by well-known anti-abuse mechanisms:
 - starting a media session is available only in secure contexts,
 - the user has to explicitly allow PTZ through a permission prompt.
 
+The page must be visible when the website updates camera pan, tilt, and zoom
+with `applyConstraints()`.
```

Comment:
else it should fail with `SecurityError`.

Reply:
Fixed. Thanks!